Misc/NEWS.d/next/Library/2023-02-19-01-49-46.gh-issue-102038.n3if3D.rst

```diff
@@ -0,0 +1 @@
+Skip a ``stat`` in :mod:`site` if we have already found a ``pyvenv.cfg``
```

Comment:
With `stat` you mean the `os.path.isfile` check that is skipped? I am unfamiliar with the term, but from the news entry alone I did not understand this.

Reply:
Yeah, an unnecessary os.path.isfile is skipped. "stat" here comes from the name of the syscall: https://en.wikipedia.org/wiki/Stat_(system_call)